src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -670,4 +670,7 @@
   <data name="PX1095Title_PXDBScalar" xml:space="preserve">
     <value>A field with the PXDBScalar attribute must have an unbound type attribute, such as PXDate, PXDecimal</value>
   </data>
+  <data name="PX1096Title" xml:space="preserve">
+    <value>PXOverride method signature must match the overriden method</value>
```

Comment:
@EAndrosova please check the error message for the new diagnostic

Reply:
Please replace with the following:
The signature of the method with the PXOverride attribute must match the overriden method

Reply:
Updated. Thank you.